site2/docs/concepts-messaging.md

```diff
@@ -273,13 +273,33 @@ Consumer<byte[]> consumer = pulsarClient.newConsumer(Schema.BYTES)
                 .subscribe();
 ```
 
+Retry letter message contains some special property，automatically created by client.
+
+special property | Description
+:--------------------|:-----------
+`REAL_TOPIC` | The real topic name.
+`ORIGIN_MESSAGE_ID` | The origin message id, it will be crucial for message tracking.
```

Comment:
```suggestion
`ORIGIN_MESSAGE_ID` | The origin message ID. It is crucial for message tracking.
```
- Write in the simple present tense as much as possible if you are covering facts that were, are, and forever shall be true. Details see https://docs.google.com/document/d/1lc5j4RtuLIzlEYCBo97AC8-U_3Erzs_lxpkDuseU0n4/edit#bookmark=id.e8uqh1awkcnp

- Capitalize proper nouns correctly. https://docs.google.com/document/d/1lc5j4RtuLIzlEYCBo97AC8-U_3Erzs_lxpkDuseU0n4/edit#bookmark=id.je5thtkj42cb